client/src/components/More.jsx

```diff
@@ -105,6 +106,10 @@ const More = () => {
       });
   };
 
+  const navigateTo = href => {
+    history.push(href);
+  }
+
```

Comment:
Any reasoning for switching to `useHistory`?

Reply:
Yeah. Because of we using Button component for these elements and for JSX react-router requires use Link component, as far as I know there are only two ways how to fix issue: restyle Link component to make it look like Button or just use Button component and on click programmaticaly redirect user on specified page. The second option is pretty natural and has no difference from using Link in terms of redirecting user on another page. It's just don't affects on styles unlike Link component.

Reply:
Great - makes a lot of sense!